README.md

```diff
@@ -37,7 +37,11 @@ to the contract essentially means that the account that owns the scope has reque
 data within the scope and mark the scope as a "classified asset," indicating its authenticity as a properly-formed
 object that the Provenance Blockchain recognizes.  The process of a successful onboarding will create and store an
 [AssetScopeAttribute](src/core/types/asset_scope_attribute.rs) struct value, serialized as JSON in a [Provenance Metadata Attribute](https://docs.provenance.io/modules/account)
-on the scope.
+on the scope.  During the onboarding process, the onboarding account has the option of choosing whether or not to trust
```

Comment:
This can probably just go back to what it was before, along with the other pieces in here not about the new `query_fee_payments` or the removal of the bind contract alias.  Another :rip:

Reply:
ripped out